src/main/java/org/dependencytrack/resources/v1/TagResource.java

```diff
@@ -146,6 +146,36 @@ public Response deleteTags(
         return Response.noContent().build();
     }
 
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Creates one or more tags.",
+            description = "<p>Requires permission <strong>TAG_MANAGEMENT</strong></p>\n"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "201",
+                    description = "Tags created successfully."
+            ),
+            @ApiResponse(
+                    responseCode = "400",
+                    description = "Operation failed",
+                    content = @Content(schema = @Schema(implementation = TagOperationProblemDetails.class), mediaType = ProblemDetails.MEDIA_TYPE_JSON)
+            )
```

Comment:
`qm.createTags` will not throw a `TagOperationFailedException` so this particular response will never be returned. Best to omit this `@ApiResponse` definition to avoid confusion.

Reply:
I have copied this from another endpoint. I am hesitant to change the other endpoint which specified this as a response, should I do that?

Reply:
Depends on what other endpoint you mean. If they declare this error response, their logic *should* be throwing a `TagOperationFailedException`, which would then be [mapped to `TagOperationProblemDetails`](https://github.com/DependencyTrack/dependency-track/blob/master/src/main/java/org/dependencytrack/resources/v1/exception/TagOperationFailedExceptionMapper.java). If that is not the case, then the error declaration is wrong and could safely be removed.